{"name": "Test generate token with invalid accounts", "status": "passed", "parameters": [{"name": "account", "value": "<data_object.account.Account object at 0x000001F768250740>"}], "start": 1702454884668, "stop": 1702454886198, "uuid": "98c93088-55ce-4cd9-8db5-08f56f32b4e1", "historyId": "763a8e7ddd3b12788ae3ab21f0a0cf12", "testCaseId": "573dcf4d17f28e829fe2a39fec07d16d", "fullName": "testcases.api.test_generate_token#test_generate_token_unsuccessfully_with_invalid_account", "labels": [{"name": "parentSuite", "value": "API Test"}, {"name": "suite", "value": "Test Generate Token"}, {"name": "subSuite", "value": "TestGenerateToken"}, {"name": "host", "value": "VNNOT02222"}, {"name": "thread", "value": "3328-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api.test_generate_token"}]}